assets/js/blocks/cart-checkout/checkout/style.scss

```diff
@@ -260,10 +260,6 @@
 		margin-bottom: $gap-largest;
 		order: 0;
 	}
-
-	.wc-block-checkout__main-totals {
```

Comment:
This div is now gone (no longer needed). It looks like this style rule was to ensure it flowed after checkout form on mobile - which now happens by default :)